src/templates/docker-compose.yml.j2

```diff
@@ -148,256 +137,11 @@ services:
-      - "traefik.http.routers.nextcloud.rule=Host(`cloud.{{ pinanas.domain }}`)"
-      - "traefik.http.routers.nextcloud.middlewares=nextcloud-https-redirect"
-      - "traefik.http.middlewares.nextcloud-https-redirect.redirectscheme.scheme=https"
-      - "traefik.http.services.nextcloud-secure.loadbalancer.server.scheme=https"
-      - "traefik.http.services.nextcloud-secure.loadbalancer.server.port=443"
-      - "traefik.http.middlewares.hsts.headers.stsincludesubdomains=false"
-      - "traefik.http.middlewares.hsts.headers.stspreload=true"
-      - "traefik.http.middlewares.hsts.headers.stsseconds=31536000"
-      - "traefik.http.routers.nextcloud-secure.middlewares=hsts"
-    volumes:
-      - ./nextcloud/data/crontabs:/config/crontabs
-      - ./nextcloud/data/generated-config:/config/www/nextcloud/config
-      - ./nextcloud/data/files:/data
-    restart: unless-stopped
-
-
-  #
-  ## === JELLYFIN ===
-  #
-  # Jellyfin is a Free Software Media System that puts you in control of
-  # managing and streaming your media. It is an alternative to the proprietary
-  # Emby and Plex, to provide media from a dedicated server to end-user
-  # devices via multiple apps.
-  ##
-  jellyfin:
-    image: lscr.io/linuxserver/jellyfin:10.8.7
-    container_name: jellyfin
-    networks:
-      pinanas:
-        aliases:
-          - jellyfin
-          - media
-    environment:
-      - TZ={{ pinanas.timezone }}
-      - PUID={{ PUID }}
-      - PGID={{ PGID }}
-    labels:
-      - "traefik.enable=true"
-      - "traefik.http.middlewares.jellyfin-https-redirect.redirectscheme.scheme=https"
-      - "traefik.http.routers.jellyfin-secure.entrypoints=https"
-      - "traefik.http.routers.jellyfin-secure.middlewares=authelia@docker"
-      - "traefik.http.routers.jellyfin-secure.rule=Host(`media.{{ pinanas.domain }}`)"
-      - "traefik.http.routers.jellyfin-secure.service=jellyfin"
-      - "traefik.http.routers.jellyfin-secure.tls=true"
-      - "traefik.http.routers.jellyfin.entrypoints=http"
-      - "traefik.http.routers.jellyfin.middlewares=jellyfin-https-redirect"
-      - "traefik.http.routers.jellyfin.rule=Host(`media.{{ pinanas.domain }}`)"
-      - "traefik.http.services.jellyfin.loadbalancer.server.port=8096"
-    volumes:
-      {% for configfile in ['branding.xml', 'network.xml', 'system.xml'] %}
-      - ./jellyfin/config/{{ configfile }}:/config/{{ configfile }}
-      {% endfor %}
-      - ./jellyfin/config/plugins:/config/data/plugins
-      - ./jellyfin/config/libraries/default:/config/data/root/default
-      - ./jellyfin/data/log:/config/log
-      - ./jellyfin/data/movies:/data/movies
-      - ./jellyfin/data/tvshows:/data/tvshows
-      - ./jellyfin/data/misc/metadata:/data/metadata
-      - ./jellyfin/data/misc/cache:/data/cache
-      - ./jellyfin/data/misc/databases:/config/data/data
-    restart: unless-stopped
+{% if pinanas.applications is not none -%}
+  {% macro docker_compose() -%}
```

Comment:
Check the same behaviour isn't possible without the use of a complicated macro.

Reply:
Macro allows to add "indent" when concat files and, so, avoid the need to add it in each "apps.yml" files. Maybe an other way should exit but after hours of exploring jinja2 documentation it's appear me the easiest option... Tell me if a macro is a problem or non-conventional.

Reply:
Can you have a look at https://stackoverflow.com/a/53775887/5470596? I cannot test it right now.
```jinja2
{% filter indent(width=4) %}
{% include "./sub-template.yml.j2" %}
{% endfilter %}
```